src/guides/v2.4/graphql/mutations/request-password-reset-email.md

```diff
@@ -0,0 +1,63 @@
+---
+group: graphql
+title: requestPasswordResetEmail mutation
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The `requestPasswordResetEmail` mutation triggers the password reset email by the provided email address. Use it to initiate a reset password process for the registered customer before [resetPassword]({{page.baseurl}}/graphql/mutations/reset-password.html) mutation call.
```

Comment:
Could you double-check, please, that we have the "Reset Password" related page? I don't see it neither in the mainline nor in the 2.4.1 branch. 

We have another page for changing customer password (https://devdocs.magento.com/guides/v2.4/graphql/mutations/change-customer-password.html) but I may be missing something. 

Thank you

Reply:
Yes.

Check this one - https://github.com/magento/devdocs/pull/7426

THank you, @rogyar